src/style.css

```
@@ -0,0 +1,76 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+input.search {
```

all inputs have a class search you can remove `input` from this line
```suggestion
.search {
```
in general, that's a bad idea: don't use tags, use classes for this